php-files/functions.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @return PDO
+ * Connects to database
+ */
+function databaseConnect()
+{
+    $db = new PDO('mysql:host=192.168.20.20; dbname=fish-finder', 'root', '');
+    return $db;
+}
+
+/**
+ * @param $db
+ * @return array
+ * Queries the database to get multidimensional array of all data required to display.
+ */
+function getFish($db): array
+{
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
I would set this attribute as part of your databaseConnect() function, before you return the PDO - there isn't a time that you would want to use your DB without this attribute set.